src/stores/WidgetStore.ts

```diff
@@ -207,24 +209,6 @@ export default class WidgetStore extends AsyncStoreWithClient<IState> {
             roomApps.widgets = roomApps.widgets.filter((app) => !(app.id === widgetId && app.roomId === roomId));
         }
     }
-
-    public doesRoomHaveConference(room: Room): boolean {
```

Comment:
If public functions are removed, does this PR than introduce a breaking change?

Reply:
If this was a real SDK then yes, but as per the README:

> As of Aug 2018, the only skin that exists is [vector-im/element-web](https://github.com/vector-im/element-web/); it and matrix-org/matrix-react-sdk **should effectively be considered as a single project** (for instance, matrix-react-sdk bugs are currently filed against vector-im/element-web rather than this project).